src/components/bulk-email-tool/bulk-email-task-manager/messages.js

```diff
@@ -0,0 +1,78 @@
+import { defineMessages } from '@edx/frontend-platform/i18n';
```

Comment:
I extracted the messages to a separate file with the addition of the tables and modal. I felt that it made the Component files a bit easier to read. I saw that this is a common practice in the learning MFE (and remember seeing it in other MFE's as well).